### DISCUSSION TITLE
字符转换数字一直报错

### DISCUSSION THREAD

**Discussion by Jennyhua8** [datawhalechina/learn-python-the-smart-way-v2#31] [提问答疑]
ame = input("你的名字：")  
print("Your name is", name, "Cheers")  
  
x = input("输入一个数字：")  
x = int(x) # 类型转换，将字符串转化为数字  
#x = int(input("输入一个数字："))
print(x, "的一半等于", x/2) # 对味啦

<img width="753" alt="截屏2023-12-12 20 39 59" src="https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/153477645/c931415d-5281-4ad5-b7a5-a21d8db1baec">

**Comment by anine09** [comment]
hi，@Jennyhua8 这个问题不符合我们的提问规范，我们无法提供对应的帮助，在提问之前请仔细阅读[提问规则](https://datawhalechina.github.io/learn-python-the-smart-way-v2/Question/question/)
